tom_registration/registration_flows/open/views.py

```diff
@@ -21,6 +22,7 @@ class OpenRegistrationView(CreateView):
     template_name = 'tom_registration/register_user.html'
     success_url = reverse_lazy(settings.TOM_REGISTRATION.get('REGISTRATION_REDIRECT_PATTERN', ''))
     form_class = OpenRegistrationForm
+    extra_context = {'version': __version__}
```

Comment:
nice -- I didn't know you could add to the context this way. Now I see [it](https://ccbv.co.uk/projects/Django/5.0/django.views.generic.base/ContextMixin/#get_context_data)